archive/app/controllers/ArchiveController.scala

```diff
@@ -133,33 +130,25 @@ class ArchiveController(dynamoDB: DynamoDB) extends Controller with Logging with
   }
 
   private def redirectTo(path: String, identifier: String)(implicit request: RequestHeader): Result = {
-    log.info(s"301, $identifier, ${RequestLog(request)}")
-    Cached(300)(WithoutRevalidationResult(Redirect(s"http://$path", 301)))
+    log.info(s"Archive $redirectHttpStatus, \"$identifier\" redirect to $path")
+    Cached(CacheTime.ArchiveRedirect)(WithoutRevalidationResult(Redirect(s"http://$path", redirectHttpStatus)))
   }
 
-  private def logDestination(path: String, msg: String, destination: String) {
-    log.info(s"Destination: $msg : $path -> $destination")
-  }
-
-  private def log404(request: Request[AnyContent]) =
+  private def log404(request: Request[AnyContent]) = {
+    log.warn(s"Archive returned 404 for path: ${request.path}")
```

Comment:
Kept + improved this log message since it's the only way to see that archive itself also returned 404